plugins.json

```diff
@@ -989,9 +1045,11 @@
       }
     },
     "enterprise": false,
+    "platforms": {},
```

Comment:
That is surprising. There should be one for this plugin. Could you please check why there isn't?